integration-tests/testconfig/default.toml

```diff
@@ -7,16 +12,37 @@ log_producer_timeout="10s"
 log_producer_retry_limit=10
 
 [Network]
-selected_networks=["simulated"]
-
-[PrivateEthereumNetwork]
-consensus_type="pow"
-execution_layer="geth"
-
-[PrivateEthereumNetwork.EthereumChainConfig]
-seconds_per_slot=3
-slots_per_epoch=2
-genesis_delay=15
-validator_count=4
-chain_id=1337
-addresses_to_fund=["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"]
\ No newline at end of file
+selected_networks=["SIMULATED"] # Not needed for Solana but mandatory from CTF (do not change)
+
+[Network.RpcHttpUrls]
+simulated = ["http://127.0.0.1"] # Not needed for Solana but mandatory from CTF (do not change)
+
+[Network.RpcWsUrls]
+simulated = ["wss://127.0.0.1"] # Not needed for Solana but mandatory from CTF (do not change)
+
+# Testnet program ID's
+[SolanaConfig]
+ocr2_program_id = "cjg3oHmg9uuPsP8D6g29NWvhySJkdYdAo9D25PRbKXJ"
+access_controller_program_id = "9xi644bRR8birboDGdTiwBq3C7VEeR7VuamRYYXCubUW"
+store_program_id = "HEvSKofvBgfaexv23kMabbYqxasxU3mQ4ibBMEmJWHny"
+link_token_address = "7CF1GrsZsny5j9JESPj98MdYVZK38RE8ZpmTEMwECK4c"
+vault_address = "FdM4dnhVpFQfjPqNG6LEfzArhuGhUjtidYu89qtGwJCS"
+secret="thisisatestingonlysecret"
+
+[Common]
+rpc_url = "https://api.devnet.solana.com"
+ws_url = "wss://api.devnet.solana.com/"
+internal_docker_repo = "public.ecr.aws/chainlink"
+inside_k8 = false 
+network = "localnet"
```

Comment:
are these just placeholder values?

the network is `localnet` but the rpc/ws are for public devnet

Reply:
Yes, just so they are set for people to know them easier, they are only taken into account if devnet is specified